client-app/src/desktop/common/Wrapper.scss

```diff
@@ -28,4 +28,8 @@
       }
     }
   }
+
+  .toolbox-wrapper-sourcecode {
+    padding: 10px;
```

Comment:
Let's use one of our standard hoist padding vars here, to keep it consistent with any future layout changes (e.g. `padding: var(--xh-pad-px);`

Reply:
good idea!